pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java

```diff
@@ -25,7 +25,7 @@ public BuildDriver getBuildDriver(BuildType buildType) throws CoreException {
             }
         }
 
-        throw new CoreException("No build driver available for " + buildType + " build type.");
+        throw new CoreException("No build driver available for " + buildType + " build type."); //TODO create test to make sure exception is properly handled
```

Comment:
This is opening wider discussion that is out of scope of this PR, do we allow TODOs in commited code? Maybe we need a Jira to track the discussion ;)